sunpy/instr/tests/test_rhessi.py

```diff
@@ -136,7 +137,7 @@ def test_parse_observing_summary_dbase_file_mock():
     """
 
     # We need to mock this test differently for < 3.7.2.
-    if sys.version_info[2] < 2 or sys.version_info[1] < 7:
+    if LooseVersion(sys.version) < LooseVersion("3.7.2"):
```

Comment:
`sys.version` is the whole date and stuff from when it was built... but I think this works fine, anyway.

Reply:
I did try passing in your one and it broke Loose and Strict.